<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>WdgPaletteListWidget</class>
 <widget class="QWidget" name="WdgPaletteListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>496</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="viewPalette" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="ctrlLine" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="bnAdd">
        <property name="icon">
         <iconset theme="add">
          <normaloff>.</normaloff>
          .
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="bnExport">
        <property name="icon">
         <iconset theme="document-export">
          <normaloff>.</normaloff>
          .
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="bnImport">
        <property name="icon">
         <iconset theme="document-open">
          <normaloff>.</normaloff>
          .
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="bnRemove">
        <property name="icon">
         <iconset theme="list-remove">
          <normaloff>.</normaloff>
          .
         </iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>